client/repolist.c

```diff
@@ -726,33 +726,34 @@ TDNFRepoListFinalize(
     /* There could be overrides to enable/disable
        repo such as cmdline args, api overrides */
     for (cn = pTdnf->pArgs->cn_setopts->first_child; cn; cn = cn->next) {
-        if(strcmp(cn->name, "enablerepo") == 0) {
-            dwError = TDNFAlterRepoState(
-                          pTdnf->pRepos,
-                          1,
-                          cn->value);
-        }
-        else if(strcmp(cn->name, "disablerepo") == 0) {
-            dwError = TDNFAlterRepoState(
-                          pTdnf->pRepos,
-                          0,
-                          cn->value);
-        }
-        else if((strcmp(cn->name, "repo") == 0) ||
-                (strcmp(cn->name, "repoid") == 0)) {
-            if (!nRepoidSeen)
-            {
-                dwError = TDNFAlterRepoState(
-                              pTdnf->pRepos, 0, "*");
-                BAIL_ON_TDNF_ERROR(dwError);
-                nRepoidSeen = 1;
+        int isRepoFlag = -1;
```

Comment:
Nit: `isRepoFlag` is a confusing name, please rename to something like `isEnabled`.